apps/registry/scripts/build-registry.ts

```
@@ -1,22 +1,67 @@
+          .replaceAll("max-w-thread", "max-w-aui-thread"),
+      )
+      .filter((k) => k)
+      .join(" "),
+  };
+});
+
+const transformClassnames = (content: string) => {
+  for (const style of tailwindStyles) {
+    if (style.key.startsWith(":where")) continue;
+
+    content = content.replaceAll(
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)